docs/static/logo.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 820 820"><path fill="none" d="M-1-1h822v822H-1z"/><g><path fill="#00b0ff" d="M110 447.8V188l225 129.9v86.6l-150-86.6v173.2l-75-43.3z"/><path fill="#0081cb" d="M335 317.9L560 188v259.8l-150 86.6-75-43.3 150-86.6v-86.6l-150 86.6v-86.6z"/><path fill="#00b0ff" d="M335 491.1v86.6l150 86.6v-86.6l-150-86.6z"/><path fill="#0081cb" d="M485 664.3l225-129.9V361.2l-75 43.3v86.6l-150 86.6v86.6zm150-346.4v-86.6l75-43.3v86.6l-75 43.3z"/></g></svg>
+<svg viewBox="0 0 820 820" xmlns="http://www.w3.org/2000/svg"><path d="m-1-1h822v822h-822z" fill="none"/><path d="m110 447.8v-259.8l225 129.9v86.6l-150-86.6v173.2z" fill="#00b0ff"/><path d="m335 317.9 225-129.9v259.8l-150 86.6-75-43.3 150-86.6v-86.6l-150 86.6z" fill="#0081cb"/><path d="m335 491.1v86.6l150 86.6v-86.6z" fill="#00b0ff"/><path d="m485 664.3 225-129.9v-173.2l-75 43.3v86.6l-150 86.6zm150-346.4v-86.6l75-43.3v86.6z" fill="#0081cb"/></svg>
```

Comment:
I'm curious, what's the diff about?

Reply:
I ran ImageOptim on the whole images folder.